reports/m1_proposal.md

```diff
@@ -0,0 +1,21 @@
+**Description of the data**
+
+The original raw data is downloaded from this URL <https://thepulseofthenation.com/#future>. This dataset is the pull result from Card Against Humanity to the nationally representative of American public about popular social concerns during time period August 1st to 15th 2018. The survey was conducted via face to face, cell phone, and land lines, and respondents' demographic info and opinion choices on survey questions were collected.
+
+The raw dataset contains 800 x 24 datapoints, including collected answers from 800 respondents about 7 demographical questions and 14 opinion questions. It contains 11% of missing data points mainly from redundant 'other' columns.
+
+In order to better understand public opinion of different demographic groups, we have done some initial data preprocessing and data engineering to get prepared for visualization. For data preprocessing, we dropped 5 irrelevant, or vacant columns. We renamed column title into brief phrase form. We added the 'political_spectrum_score' column based on numerical conversion of the 'ideaology' choice to help understand political spectrum. Further, we added the 'time_answered' column to reflect this survey period for dashboard sustainability purpose if there are more incoming surveies in future month.
```

Comment:
can we have this in bullet format maybe? we said we dropped columns , what type of columns?
we said we added two new columns : 
let's define them in bullets 
- var1: def 
- var2:def
style instead?

Reply:
For ideology score:  based on the notebook code, can we add the definition mentioning the scores for each type. Also maybe we can score only on positive scale like conservative:2, liberal: 5, moderate: 3, this way we do not get many 0's in the score. ref code: 
`ideology_to_score = {
    'Conservative': -1,  # Assuming a lower score for conservative
    'Liberal': 1,        # Assuming a higher score for liberal
    'Moderate': 0,       # Neutral score for moderate
    'DK/REF': 'DK/REF'   # Keeping DK/REF as is
} `

Reply:
can use the columns that I mentioned in my PR for dashboard design as the final ones, since there was no change to the columns in the feedback?